crawl-ref/source/dat/descript/spells.txt

```diff
@@ -201,6 +201,11 @@ related to) power.
 It is powerful enough to cancel the effects of Death's Door, although doing so
 will briefly paralyse the caster. It has no effect on the undead.
 %%%%
+Borgnjor's Vile Clutch spell
+
+Calls forth hands from corpses interred beneath the dungeon floor. Enemies
```

Comment:
It occurs to me now that this description may mislead players into thinking this only works on areas that have corpses nearby. I don't immediately have a good suggested fix, although maybe the word "ancient" could be used somehow to reinforce this is not about recent corpses?

Reply:
I suppose it's possible for someone to get confused about that, but it does say "interred beneath the dungeon floor". I'll run the addition of "ancient" by mikee, the vile clutch description master.